src/azure-cli/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -79,7 +79,7 @@ def load_arguments_eh(self, _):
             c.argument('skip_empty_archives', options_list=['--skip-empty-archives'], arg_type=get_three_state_flag(), help='A boolean value that indicates whether to Skip Empty.')
             c.argument('capture_interval_seconds', arg_group='Capture', options_list=['--capture-interval'], type=int, help='Allows you to set the frequency with which the capture to Azure Blobs will happen, value should between 60 to 900 seconds')
             c.argument('capture_size_limit_bytes', arg_group='Capture', options_list=['--capture-size-limit'], type=int, help='Defines the amount of data built up in your Event Hub before an capture operation, value should be between 10485760 to 524288000 bytes')
-            c.argument('destination_name', arg_group='Capture-Destination', help='Name for capture destination')
+            c.argument('destination_name', arg_group='Capture-Destination', help='Name for capture destination, EventHubArchive.AzureBlockBlob or EventHubArchive.AzureDataLake.')
```

Comment:
currently we are not supporting 'EventHubArchive.AzureDataLake' in clients, its a swagger issue.  we will need to remove Datalake part